layout/theme.liquid

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Procurement portal</title>
+  <base href="/">
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" type="image/x-icon" href="favicon.ico">
+    <script>
+        var BASE_URL = '{{ "/" | full_url }}';
+		var BASE_URL2 = '{{ '/' | absolute_url }}';
+    </script>
+
+</head>
+<body>
+
+    
+
+	
+<app-root>Loading...</app-root>
+<script type="text/javascript" src="{{ 'bundle/polyfills.js' | static_asset_url }}">  </script>
```

Comment:
Need to use  | append_version liquid filter to prevent caching issues

Reply:
Created task on the board